core/src/main/java/io/questdb/griffin/engine/orderby/LimitedSizeSortedLightRecordCursorFactory.java

```diff
@@ -134,27 +134,22 @@ public void initializeLimitedSizeCursor(SqlExecutionContext executionContext, Re
         long lo = loFunction.getLong(null);
         if (lo < 0 && hiFunction == null) {
             // last N rows
-            isFirstN = false;
             // lo is negative, -5 for example
             // if we have 12 records we need to skip 12-5 = 7
             // if we have 4 records = return all of them
             // set limit to return remaining rows
             limit = -lo;
-            skipFirst = skipLast = 0;
         } else if (lo > -1 && hiFunction == null) {
             // first N rows
             isFirstN = true;
             limit = lo;
-            skipFirst = skipLast = 0;
         } else {
             // at this stage we also have 'hi'
             long hi = hiFunction.getLong(null);
             if (lo < 0) {
                 // right, here we are looking for something like -10,-5 five rows away from tail
                 if (lo < hi) {
-                    isFirstN = false;
                     limit = -lo;
-                    skipFirst = 0;
                     skipLast = Math.max(-hi, 0);
                     //}
```

Comment:
nit: this line looks like a leftover from one of previous PRs

Reply:
I deliberately removed some vars there because intellij was complaining heavily. These changes didn't change outcome of tests though.

Reply:
I know. This comment is about `//}` line which doesn't seem to be a helpful comment.